hieradata_aws/class/staging/jenkins.yaml

```diff
@@ -44,6 +54,9 @@ govuk_jenkins::config::user_permissions:
     permissions:
       - 'hudson.model.Item.Build'
       - 'hudson.model.Item.Read'
+  -
+    user: 'alphagov*GOV.UK Production Deploy'
+    permissions: *jenkins_admin_permission_list
```

Comment:
I think here we need `jenkins_deploy_permission_list`? But I might be wrong

Reply:
Thanks yes you're right!